<ui version="4.0" >
 <class>AdditionalEditPage</class>
 <widget class="QWidget" name="AdditionalEditPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>-</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="GroupBox1" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Additional Properties</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="synonym_label" >
          <property name="text" >
           <string>S&amp;ynonym:</string>
          </property>
          <property name="buddy" >
           <cstring>synonymLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="synonymLineEdit" >
          <property name="toolTip" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="antonym_label" >
          <property name="text" >
           <string>Ant&amp;onym:</string>
          </property>
          <property name="buddy" >
           <cstring>antonymLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="antonymLineEdit" />
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="examp_label" >
          <property name="text" >
           <string>E&amp;xample:</string>
          </property>
          <property name="buddy" >
           <cstring>exampleLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="exampleLineEdit" />
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="remark_label" >
          <property name="text" >
           <string>&amp;Comment:</string>
          </property>
          <property name="buddy" >
           <cstring>commentLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLineEdit" name="commentLineEdit" />
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="para_label" >
          <property name="text" >
           <string>&amp;Paraphrase:</string>
          </property>
          <property name="buddy" >
           <cstring>paraphraseLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QLineEdit" name="paraphraseLineEdit" />
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="imageLabel" >
          <property name="text" >
           <string>Image:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="KUrlRequester" name="imageUrlRequester" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="audioLabel" >
          <property name="text" >
           <string>Audio:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="KUrlRequester" name="audioUrlRequester" />
        </item>
        <item row="1" column="2" >
         <widget class="QPushButton" name="audioPlayButton" >
          <property name="text" >
           <string>Play</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>410</width>
          <height>51</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
